src/main/java/com/google/sps/servlets/UserStatusServlet.java

```diff
@@ -39,7 +39,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     JsonObject userInfo = new JsonObject();
     userInfo.addProperty("isUserLoggedIn", UserHelper.isUserLoggedIn());
     if (UserHelper.isUserLoggedIn()) {
-        System.out.println("User is logged in");
+        System.out.println("Return user profile info");
```

Comment:
why'd you change what it prints out? doesn't the original make more sense?

Reply:
I'm actually gonna delete this whole file so I won't think about this question too much.